README.md

```diff
@@ -2,3 +2,12 @@
 Interactive Classroom Mangement Tool using Raspberry Pi, Python, and NeoPixels
 
 [Formbar Project Shortcut](https://github.com/csmith1188/formbar/projects/11)
+
+Used as a system in which teachers can better interact with students, and manage thier classroom effectively
+Worked on by a highschool teacher and his students, this project hopes to reach further than most others
+This is demonstrated by the students hard work and dilligent learning.
+
+Guides on how to use the formbar:
+[Formbar Project Shortcut](https://github.com/csmith1188/formbar/projects/11)
+(https://github.com/csmith1188/formbar/wiki)
+(https://github.com/csmith1188/formbar/wiki/forbius-related-schitzophreic-ramblings)
```

Comment:
Can we add a bit more. Like a short summary on the steps to set up and use?

Example:
"Step one, clone repo (link to guide on doing this)
Step two, install and configure (link to guide on doing this)
Step three, create teacher account (link to guide)"

etc.